.github/ISSUE_TEMPLATE/release-check-list.md

```diff
@@ -0,0 +1,69 @@
+- [ ] - 5. Update Kata Containers to use the latest attestation-agent
+        * https://github.com/kata-containers/kata-containers/blob/CCv0/versions.yaml
+          * Change the version
+- [ ] - 6. Update Kata Containers to use the latest td-shim
+        * https://github.com/kata-containers/kata-containers/blob/CCv0/versions.yaml
+          * Change the version
+- [ ] - 7. Check if there are new changes in the pre install payload script
+        * https://github.com/confidential-containers/operator/tree/main/install/pre-install-payload
+          * The last commit there must match what's in the following files as preInstall / postUninstall image
+            * Enclave CC: https://github.com/confidential-containers/operator/blob/main/config/samples/enclave-cc/base/ccruntime-enclave-cc.yaml
+            * Kata Containers:
+              * Note that for Kata Containers, we're looking for the newTag, below the quay.io/confidential-containers/container-engine-for-cc-payload image
+              * s390x: https://github.com/confidential-containers/operator/blob/main/config/samples/ccruntime/s390x/kustomizat
+              * x86_64: https://github.com/confidential-containers/operator/blob/main/config/samples/ccruntime/default/kustomization.yaml
+- [ ] - 8. Ensure the Operator is using the latest CI builds and that the Operator tests are passsing
+        * Enclave CC:
+          * SIM: https://github.com/confidential-containers/operator/blob/main/config/samples/enclave-cc/sim/kustomization.yaml
+          * HW: https://github.com/confidential-containers/operator/blob/main/config/samples/enclave-cc/base/ccruntime-enclave-cc.yaml
+          * Note that we need the quay.io/confidential-containers/runtime-payload-ci registry and enclave-cc-{SIM,HW}-latest tags
+        * Kata Containers:
+          * s390x: https://github.com/confidential-containers/operator/blob/main/config/samples/ccruntime/s390x/kustomization.yaml
+          * x86_64: https://github.com/confidential-containers/operator/blob/main/config/samples/ccruntime/default/kustomization.yaml
+          * Note that we need the quay.io/confidential-containers/runtime-payload-ci registry and kata-containers-latest tag
+- [ ] - 9. Cut an ocicrypt-rs v<TARGET_RELEASE> release, if changes happened in the project
+- [ ] - 10. Cut an attestation-agent v<TARGET_RELEASE>, if changes happened in the project
+- [ ] - 11. Cut an image-rs v<TARGET_RELEASE> release, using the latest release of:
+        * ocicrypt-rs (redo step 1, but now using v<TARGET_RELEASE>)
+        * attestation-agent (redo step 2, but now using v<TARGET_RELEASE>)
+- [ ] - 12. Cut a td-shim v<TARGET_RELEASE> release, if changes happened in the project
+- [ ] - 13. Update Enclave CC to use the released version of image-rs
+        * redo step 3, but now using v<TARGET_RELEASE>
+- [ ] - 14. Update the operator to use the images generated from the latest commit of both Kata Containers and Enclave CC
```

Comment:
Do we need to also redo step 4 and 5 using the released versions of image-rs and the attestation-agent?

Reply:
We do, fixed!